future_challenges_for_RSEng.tex

```diff
@@ -518,7 +515,11 @@ \section{What is in this future for RSEs and where to go from here?}
 We hope that the standard of research software will have risen across the board, enabling easier reuse of prior software outputs in new research.
 We also hope that a cultural change is taking place to recognize research software as an integral part of science that it is
 widely recognized and rewarded across academia and is supported by political actors.
-To conclude, RSEs will very likely be doing different work
+\section{Conclusion}
+We have tried to elaborate on the state on the future ten years from now.
```

Comment:
The previous section starts with "It is hard to conclude", but it is not the "Conclusion" section. I would make it:

```diff
- It is hard to conclude a piece written about the future, but we have tried to list
+ In the previous sections, we have tried to list
```

(cannot add a GitHub comment in that line)